content/projects/nav-pi/index.md

```diff
@@ -14,14 +16,69 @@ show_title_text: false
 in_progress: false
 completed: true
 show_on_roadmap: true
-show_on_projects: false
-allow_click_through: false
+show_on_projects: true
+allow_click_through: true
 percent_complete: 100
 github_url: "https://github.com/Encrypt-S/navpi"
 project_url: "https://store.navcoin.org/product/navpi-stakebox"
 reddit_url: ""
 twitter_url: ""
-card_color: ""
-font_color: ""
-weight: 0
+card_color: "#7d5ab5"
+font_color: "#fff"
+weight: 100
 ---
+
+{{< project_header
+    title="NavPi Stakebox"
+    subhead=""
+    gradient=""
+    text="All you need to do is plug it in and send it NAV to start earning Nav Coin Stake Rewards. NavPi is a Raspberry Pi device that helps to run the blockchain at low cost. It runs the NavCoin daemon and a web wallet."
+    captionPadding="100px"
+    imgSrc="navpi3.jpg"
+    imgHeight="300px"
+    imgPadding="50px"
+    bgColor="#FFF"
+    fontColor="#333">}}
+
+{{< left_image_section
+    title="Includes"
+    imgSrc="navpi-clear.jpg"
+    imgHeight="300px"
+    imgPadding="30px"
+    bgColor="#fff"
+    linkBgColor="#523971"
+    btn1Link="https://navpi.navcoinstore.com/product/navpi-stakebox/"
+    btn1Txt="Purchase a NavPi StakeBox"
+    gradient=""
+    fontColor="#333">}}
+
+    - Raspberry Pi v3<br>
+    - 16GB SD Card<br>
+    - Plastic Branded Enclosure (Black, White or Transparent)<br>
+    - NavPi StakeBox Software
+
+{{< /left_image_section >}}
+
+{{< project_links
+    title="Get NavPi"
+    subtitle="This software turns your Raspberry Pi into a low cost and convenient way to help maintain the network."
+    fontColor="#FFF"
+    subtitleFontColor="#FFF"
+    bgColor="#7d5ab5"
+    linkBgColor="#523971"
+    gradient=""
+    feature1Img="/images/icons/nav_purple-pi-large.png"
+    feature1Title="NavPi v1.0.8"
+    feature1Link="https://s3-ap-southeast-1.amazonaws.com/org.navcoin.navpi/navpi_1.0.8.img.torrent"
+    feature1ExtraSubtitle="Torrent Magnet"
```

Comment:
These links don't seem to be coming through on the page, might be a typo in the var name